brave/brave-core
# Implement Tor client updater as component extension

Fixes brave/brave-browser#724

## Submitter Checklist:


## Test Plan:

- Extension download and installation
  - Start with a clean profile
  - Launch Brave
  - Visit chrome://components
  - Verify that Tor Client Updater is listed and version number is not `0.0.0.0`
  - Verify that name of component contains platform identifier (Windows/Mac/Linux)

- Command-line switch to disable extension
  - Start with a clean profile
  - Launch Brave with `--disable-tor-client-updater-extension`
  - Visit chrome://components and verify that Tor Client Updater is not listed

## Reviewer Checklist:
